src/config/config.py

```diff
@@ -0,0 +1,109 @@
+"""Module to handle configuration application configuration parameters."""
+import configparser
+import logging
+from os.path import expanduser, isfile
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+config_file = "config.ini"
```

Comment:
```py
CONFIG_FILE_NAME = "config.ini"
```
If we want to indicate to dev tools it is a const, we can use `Final` from typing as [type](https://docs.python.org/3/library/typing.html?highlight=typing#typing.Final).

```py
CONFIG_FILE_NAME: Final[str] = "config.ini"
```